spec/directions/bake_toc_spec.rb

```diff
@@ -344,6 +347,8 @@
               <ol class="os-unit">
                 <li class="os-toc-unit-page" cnx-archive-shortid="" cnx-archive-uri="p2">
                   <a href="#p2">
+                    <span class="os-number"><span class="os-part-text">Unit </span>1</span>
```

Comment:
is the os-part-text supposed to be inside the os-number span like that? It seems a little unusual

Reply:
@LostMojo as we discussed on slack this part of toc was already there originally created by JP. :slightly_smiling_face: